app/controllers/admin/comments_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Admin
+  class CommentsController < Admin::BaseController
+    load_and_authorize_resource :conference, find_by: :short_title
```

Comment:
https://github.com/CanCanCommunity/cancancan/wiki/Nested-Resources

Even though this line is technically correct, essentially it will never run because we have no variable conference_id set, as we are navigating to /admin/comments not in admin/conference/comments.